src/miragevpn.mli

```diff
@@ -217,11 +217,6 @@ module Config : sig
   val eq : eq
   (** [eq] is an implementation of [cmp] for use with [{!equal} cmp t t2] *)
 
-  val client_generate_connect_options :
-    t -> (string, [> `Msg of string ]) result
-  (** Exports the excerpts from the client configuration sent to the server
-      when the client initially connects. *)
-
```

Comment:
This function seems very internal to me and I don't know why it was exposed in the first place - so let's not expose it anymore!